mkosi/config.py

```diff
@@ -2641,7 +2688,11 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         section="Content",
         parse=config_parse_root_password,
         paths=("mkosi.rootpw",),
-        path_read_text=True,
+        path_handler=config_make_path_handler(
+            handler=config_path_read,
```

Comment:
Let's make this executable as well, seems like it could be useful as well. Frankly, I wouldn't even allow configuring this. If we're reading the contents immediately during config parsing, then I think it'll always make sense to execute the file if it is executable to get the contents that way. So I'd remove the concept of a handler and always do the equivalent of run_or_read().